app/src/main/java/com/example/yard/activities/MainActivity.java

```diff
@@ -77,7 +85,59 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
         Intent intent = new Intent(this, LoginActivity.class);
         startActivity(intent);
         return true;
+      case R.id.btnDelete:
+        deleteUser();
     }
     return super.onOptionsItemSelected(item);
   }
+
+  private void deleteUser() {
+    AlertDialog.Builder builder = new AlertDialog.Builder(this);
+    builder
+        .setTitle("DELETE ACCOUNT")
```

Comment:
Suggest to learn Android resources files, moving these to string.xml.

You can do the changes in another PR.